CHANGELOG.rst

```diff
@@ -3,6 +3,12 @@ Version history / changelog
 
 From version 2.0.0, turbodbc adapts semantic versioning.
 
+Version 3.4.0
+-------------
+
+* Support Arrow 0.16.x and 0.17.x, require at least 0.15.x
+* Minimal NumPy version was bumped to 1.6
```

Comment:
Typo: This should be `1.16`

What's driving the minimal version of numpy here?

Reply:
This version is also supported for Python 3.8, 1.14 isn't. Supporting 1.14 for olders would be possible but some more bash trickery but I'd rather follow NEP-29 for NumPy versions too instead of doing extra work to support old NumPy versions.

```suggestion
* Minimal NumPy version was bumped to 1.16
```

Reply:
According to NEP-29 we should still stay on 1.15 until July but that's fine for me.